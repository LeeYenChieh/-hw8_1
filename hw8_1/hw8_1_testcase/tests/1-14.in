sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL 6GTe4Nca0U6cRPu6IimETwEACia0OSG3ToCo6IhbkBbqHUR
sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL display
sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL un3d8RSELJr3Lady6kBWzc7dmQnMAXRsAElTte63iyPwHXT
sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL ZNNe481VRB0ThdoBv3pSg4meY9g91DIYBhNm4skZtOPkWDwAzgAlWo18j
sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL display
sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL LbqxquCjFUa9Xohrsy
sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL add sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL display
sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL add sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL add sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL vjyTDeWIj3HmR61NgNrplYbnoFwJIczPA4JCitACcY
sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL add scrollBar
sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL 1gNLkd
sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL display
