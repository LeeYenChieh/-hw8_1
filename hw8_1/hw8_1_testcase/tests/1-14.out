6GTe4Nca0U6cRPu6IimETwEACia0OSG3ToCo6IhbkBbqHUR 
ZNNe481VRB0ThdoBv3pSg4meY9g91DIYBhNm4skZtOPkWDwAzgAlWo18j 
LbqxquCjFUa9Xohrsy sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL 
1gNLkd sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL sOTym7Cb2TQlAF5vxRMUkS106UBtkZveblkYbJ18U4iTYEWSOL vjyTDeWIj3HmR61NgNrplYbnoFwJIczPA4JCitACcY scrollBar 
